main/src/main/java/org/mobilitydata/gtfsvalidator/util/shape/StopToShapeMatcherSettings.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.util.shape;
+
+/** Thresholds for {@code StopToShapeMatcher}. */
+public class StopToShapeMatcherSettings {
+
+  public static final double DEFAULT_MAX_DISTANCE_FROM_STOP_TO_SHAPE_IN_METERS = 200.0;
```

Comment:
Default should be 100 as suggested by the GTFS Best Practices (see https://github.com/MobilityData/GTFS_Schedule_Best-Practices/blob/master/en/shapes.md)

Reply:
OK.